app/src/main/res/values/strings.xml

```diff
@@ -24,7 +24,11 @@
   <string name="state_end_exploration_button">RETURN TO TOPIC</string>
   <string name="state_learn_again_button">LEARN AGAIN</string>
   <string name="topic_overview_see_more">See More</string>
-  <string name="topic_download_text">(%s MB)</string>
+  <string name="topic_download_text"> (%s MB)</string>
+  <string name="overview">OVERVIEW</string>
```

Comment:
We should be capitalizing via textAllCaps via the theme instead of in the string file (see https://stackoverflow.com/questions/31295116/androidtextallcaps-false-not-working-for-tablayout-design-support for a reference, though in that they are trying to do the opposite--remove the capitalization).

Reply:
suggestions where not working so used app:tabTextAppearance="@android:style/TextAppearance.DeviceDefault.Widget.TabWidget"